### ISSUE TITLE
1d dropout not 2d dropout

### ISSUE THREAD

**Issue by XiaoCode-er** [cagbal/Skeleton-Based-Action-Recognition-Papers-and-Notes#1]
In HCN, they use dropout with 0.5 probability , not 2d dropout. The accuracy of action recognition will decline when i use 2d dropout for training in my experiment.

**Comment by cagbal** [comment]
In [this code](https://github.com/huguyuehuhu/HCN-pytorch), they use 2D dropout; that's why, I said like this, but you are true, they did not mention any 2D in the paper. At least, I could not find it.

**Comment by cagbal** [comment]
Thanks again for the comment 👍  I missed it badly 😞 This issue will be resolved with the PR #2.
